api/commands/build.ts

```diff
@@ -81,15 +81,7 @@ export class Builder {
 
     const tasks = t.disabled
       ? emptyTasks
```

Comment:
cleanup: let's just return `? [] as dataform.IExecutionTask[]` and remove `emptyTasks`